### ISSUE TITLE
Space inside brackets cop raises a false positive

### ISSUE THREAD

**Issue by dblock** [rubocop/rubocop#645]
I'll work on a simpler repro, but ...

``` ruby
      it "doesn't allow more than one parameter" do
        expect {
          subject.params do
            requires(:items, desc: 'Foo') do
              requires :key
            end
          end
        }.to raise_error ArgumentError
      end

```

```
spec/grape/validations_spec.rb:106:17: C: Space inside empty braces detected.
        expect {
```

Autocorrect trashes the code:

```
       it "doesn't allow more than one parameter" do
-        expect {
-          subject.params do
-            requires(:items, desc: 'Foo') do
-              requires :key
-            end
-          end
-        }.to raise_error ArgumentError
+        expect {}.to raise_error ArgumentError
       end
```
